libs/components/forms/src/lib/modules/checkbox/checkbox.component.spec.ts

```diff
@@ -418,35 +430,40 @@ describe('Checkbox component', () => {
     let testComponent: CheckboxWithChangeEventComponent;
     let inputElement: HTMLInputElement;
 
-    beforeEach(async(() => {
+    beforeEach(async () => {
       fixture = TestBed.createComponent(CheckboxWithChangeEventComponent);
 
       fixture.detectChanges();
 
-      fixture.whenStable().then(() => {
-        checkboxDebugElement = fixture.debugElement.query(
-          By.directive(SkyCheckboxComponent)
-        );
-        checkboxNativeElement = checkboxDebugElement.nativeElement;
-        checkboxInstance = checkboxDebugElement.componentInstance;
-        testComponent = fixture.debugElement.componentInstance;
-        inputElement = checkboxNativeElement.querySelector('input');
-      });
-    }));
+      await fixture.whenStable();
+      checkboxDebugElement = fixture.debugElement.query(
+        By.directive(SkyCheckboxComponent)
+      );
+      checkboxNativeElement = checkboxDebugElement.nativeElement;
+      checkboxInstance = checkboxDebugElement.componentInstance;
+      testComponent = fixture.debugElement.componentInstance;
 
-    it('should call not call the change event when the checkbox is not interacted with', async(() => {
+      const input = checkboxNativeElement.querySelector('input');
+
+      if (!input) {
```

Comment:
This pattern was mentioned by Paul in another PR, but we should probably use the `?` instead of throwing an error in the spec.

Reply:
Done